### PR TITLE
Implement overall layout of webapp 

### PR DESCRIPTION
**Pull request by lin8231**
* Removed placeholder code 
* Created container divs for page sections 
* Created CSS elements for page corner triangles + arrows

Concern: 
* Need to refactor in the future for responsiveness

### CODE REVIEW

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -1,9 +1,83 @@
...
+      bottom: 0;
+      right: 0;
+}
+
+
+
+.upperTriangle{
+  width: 0;
+  height: 0;
+  border-top: 200px solid #F9F9F9 ;
```
Comment (texm) [inline portfolio/src/main/webapp/style.css:57]: nit: remove space before semicolon

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -1,9 +1,83 @@
...
+      border-bottom: 200px solid #F9F9F9 ;
+      border-left: 200px solid transparent;
+      position: absolute;
+      bottom: 0;
+      right: 0;
+}
+
+
+
+.upperTriangle{
```
Comment (texm) [inline portfolio/src/main/webapp/style.css:54]: nit: add space before curly bracket

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -1,9 +1,83 @@
...
+
+#linksPage {
+
+}
+
+
+.lowerTriangle{
+      width: 0;
+      height: 0;
+      border-bottom: 200px solid #F9F9F9 ;
```
Comment (texm) [inline portfolio/src/main/webapp/style.css:45]: nit: remove space before semicolon

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -1,9 +1,83 @@
...
+#undecidedPage {
+
+}
+
+#linksPage {
+
+}
+
+
+.lowerTriangle{
```
Comment (texm) [inline portfolio/src/main/webapp/style.css:42]: nit: add space before curly bracket

**Review by texm — COMMENTED** [review]
LGTM, just nitpicky stuff

**Comment by lin8231** [comment]
Thanks for your feedback! Issues have been amended.

**Review by leonnemets — APPROVED** [review]

File: portfolio/src/main/webapp/style.css [hunk 5]
```diff
@@ -1,12 +1,20 @@
+body {
+  margin: 0;
```
Comment (leonnemets) [inline portfolio/src/main/webapp/style.css:2]: Whats the default?

**Review by leonnemets — APPROVED** [review]

**Review by leonnemets — APPROVED** [review]

File: portfolio/src/main/webapp/style.css (hunk 5 above)
Comment (lin8231) [reply portfolio/src/main/webapp/style.css:2]: Default margin on the body element is 8px. So if not removed, would have a white border!

**Pull request merged**